core/src/main/java/org/web3j/protocol/core/methods/response/Log.java

```diff
@@ -207,4 +207,20 @@ public int hashCode() {
         result = 31 * result + (getTopics() != null ? getTopics().hashCode() : 0);
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "Log{" +
+                "removed=" + removed +
```

Comment:
The `+` need to be at the start of the line to keep Checkstyle happy.
Will merge once updated.